config/initializers/devise.rb

```diff
@@ -228,4 +228,21 @@
   #   manager.intercept_401 = false
   #   manager.default_strategies(:scope => :user).unshift :some_external_strategy
   # end
-end
+  config.saml_create_user = true
+  config.saml_update_user = true
+  config.saml_default_user_key = :email
+  config.saml_session_index_key = :session_index
+  config.saml_use_subject = true
+  config.idp_settings_adapter = nil
+  config.saml_configure do |settings|
+    settings.assertion_consumer_service_url     = ENV["ASSERTION_CONSUMER_SERVICE_URL"]
+    settings.assertion_consumer_service_binding = "urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST"
+    settings.name_identifier_format             = "urn:oasis:names:tc:SAML:2.0:nameid-format:transient"
+    settings.issuer                             = ENV["ISSUER"]
+    settings.authn_context                      = ""
+    settings.idp_slo_target_url                 = ""
+    settings.idp_sso_target_url                 = ENV["IDP_SSO_TARGET_URL"]
```

Comment:
This will vary depending upon whether the auth request is an `HTTP-POST` or `HTTP-Redirect`